klp_configure.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright (C) 2019 SUSE
+# Author: Libor Pechacek
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>.
+
+set -e
+. "$(dirname $0)"/klp_tc_functions.sh
+
+# compile-test for hrtimer API ()
+COMPILETEST_DIR=/tmp/live-patch/klp_compile_test
+mkdir -p $COMPILETEST_DIR
+cp "$SOURCE_DIR"/klp_compile_test_hrtimer.c $COMPILETEST_DIR/
+
+echo -n 'export KLP_TEST_HRTIMER_OLD='
+if klp_compile_module /tmp/live-patch/klp_compile_test/klp_compile_test_hrtimer.c > /dev/null 2>&1;
```

Comment:
$COMPILETEST_DIR could be used in the source file specification.

Reply:
Yes, that one slipped through. Fixed. Thanks!